src/main/scala/io/github/cloudify/scala.spdf/DestinationDocumentLike.scala

```diff
@@ -14,11 +17,13 @@ trait DestinationDocumentLike[-A] {
   /**
    * The destination argument to supply to `wkhtmltopdf`
    */
+  @silent
```

Comment:
Why are they necessary?

Reply:
Some methods have parameters that are not in used.

Reply:
And since those methods are APIs, I decided to suppress the warning for now.